src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/fixtures/marketplace-data-extensions.json

```
@@ -1,5 +1,5 @@
 {
-    "total": 120,
+    "count": 120,
```

```suggestion
    "total": 120,
```

The marketplace spec explicitely specify `total`

---

Sometimes it was `count` in our code, other `total`. I will harmonize with 'total' so

---

yes, but here, it's the mock for the marketplace endpoint, if we don't follow the format of the marketplace API, we will hit a wall :sweat_smile: